prime-router/src/testIntegration/resources/datatests/mappinginventory/catchall/orcobr/orc_obr-to-servicerequest_diagnosticreport-obr8-populated.fhir

```diff
@@ -147,6 +147,24 @@
       "extension" : [ {
         "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/obr-observation-request",
         "extension" : [ {
+          "url" : "OBR.7, OBR.8",
```

Comment:
todo: I think `"url" : "Field1.x, Field2.x"` is how we represent joined fields in extensions, but I need to verify this.

Reply:
This was resolved by not mapping them as a period in extension but as separate date times.

Also as a note, we have one instance of this happening with others fields. In some scenarios we set the url value to "HD.2,HD.3".